### DISCUSSION TITLE
Looking for a new maintainer for PaperWM

### DISCUSSION THREAD

**Discussion by jtaala** [paperwm/PaperWM#944] [Announcements]
Hey all,

So, I find myself with very little time these days to continue working on PaperWM.

PaperWM fits my needs perfectly, and I'm happy with the current feature set - but I don't have much time anymore to work on making it more useful for others' workflows.

As such, I'm planning on stepping back from development and (hopefully) letting others step in.

I'm still planning to work on bugs (that I can reproduce/solve) and will try to find time to do updates to make PaperWM compatible with newer version of Gnome etc.

Please reach out if you're interested in taking on PaperWM moving forward.

Regards,

Jay.

**Comment by Thesola10** [comment]
As mentioned in #980, I am preparing to take up maintainership for PaperWM and an important part of this is write access to the [GNOME Extensions](https://extensions.gnome.org/extension/6099/paperwm/) entry.

The ideal solution would be to have EGO uploading as part of a GitHub Actions process, with credentials to the EGO account/key locked as a Secret.

Right now this doesn't look like a thing EGO allows, but the issue a dedicated PaperWM account on EGO stands, as a shared secret between maintainers.

As you are the current owner of the extension on EGO, could you look into this, or transfer ownership to [my account](https://extensions.gnome.org/accounts/profile/thesola10) as a stopgap measure?

Or maybe EGO allows a third party to submit a new extension version (bad), right now I'm not sure.

**Reply by jtaala** [reply]
Thanks @Thesola10.  Sorry for the delayed response.

I had a quick look - and it doesn't seem like there's a way to transfer ownership to your account.  I'll look at changing account details (to something generic) and doing a shared secret with you.

Jay.
